tests/integration_tests/instances.py

```diff
@@ -56,8 +56,8 @@ def push_file(self, local_path, remote_path):
         self.instance.push_file(str(local_path), tmp_path)
         self.execute('mv {} {}'.format(tmp_path, str(remote_path)))
 
-    def read_from_file(self, remote_path) -> str:
-        result = self.execute('cat {}'.format(remote_path))
+    def read_from_file(self, remote_path, use_sudo=None) -> str:
```

Comment:
Let's use_sudo=True so we aren't changing types from None -> bool

Reply:
The rationale behind that is because this is default value of the `execute` method:

https://github.com/canonical/cloud-init/blob/master/tests/integration_tests/instances.py#L40

Since we are calling that method inside the `read_from_file`, I wanted to make it consistent with the `execute` method as well

Reply:
My mistake here, reviewed the existing execute approach and that aligns here.